README.md

```diff
@@ -47,7 +85,7 @@ If you want to use it in another maven project, after installing it you can use
         <dependency>
             <groupId>org.blosc</groupId>
             <artifactId>jblosc</artifactId>
-            <version>1.0</version>
+            <version>JBLOSC_VERSION_X.Y.Z</version>
```

Comment:
These are instructions for users, so better do JBLOSC_VERSION_X.Y.Z -> JBLOSC_VERSION